Parser/tokenizer.c

```diff
@@ -1378,9 +1365,8 @@ tok_get(struct tok_state *tok, char **p_start, char **p_end)
                 col++, altcol++;
             }
             else if (c == '\t') {
-                col = (col/tok->tabsize + 1) * tok->tabsize;
-                altcol = (altcol/tok->alttabsize + 1)
-                    * tok->alttabsize;
+                col = (col / tok->tabsize + 1) * tok->tabsize;
+                altcol = (altcol / ALTTABSIZE + 1) * ALTTABSIZE;
```

Comment:
This is just `altcol++`, isn't?

Reply:
I don't know well this code, so I prefer to minimize my changes to avoid any risk of regression.

It looks like the code doesn't round properly for tabsize=1. But I was too lazy to try to understand this old code. I prefer to leave it as it is if something has to change the code later.

FYI I found this code when I wanted to enable tabs/spaces warning in my new developer mode... Then I understood that the warning is always emitted and it's even an hard error...

Reply:
I recently learned a new term: [Chesterton's fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence) :smile:.

Yes, since we are not going to allow mixing spaces and tabs, `altwarning` and `alterror` can be removed. But the algorithm not always works properly (see [bpo-32053](https://bugs.python.org/issue32053)), and I think that it would be better at least to write it in simpler way.

Reply:
**Please** go ahead and continue to cleanup this code ;-) Be for my side, I'm done. As I wrote, I only cared when the syntax warning was emitted (and it's now *always* emitted).

Reply:
I would prefer to not introduce useless and misleading parameters like ALTTABSIZE.

And, to be pedantic, the syntax warning is now *never* emitted (the syntax error is emitted instead).